docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -566,10 +498,105 @@ const DebugPage: React.FunctionComponent<Props> = ({
     });
   }, [serviceType, transport, method, examplePaths]);
 
-  const [debugAlertIsOpen, setDebugAlertIsOpen] = React.useState(true);
-
   return (
-    <div>
+    <>
+      <Section>
+        <div id="debug-form">
```

Comment:
nit; I think the double `debug-form` is preventing `copy as curl`/`copy response` to work correctly in the modal form